README.md

```diff
@@ -120,23 +120,40 @@ For example, the following arguments were used to build the IoT Hub Device Updat
 
 ```sh
 # Run following commands from <projectroot> directory
-yocto_release=honister
-adu_release=honister
+yocto_release=kirkstone
+adu_release=kirkstone
+
 adu_src_uri="gitsm://github.com/azure/iot-hub-device-update"
+
 # NOTE: Must specify the specific 'branch' and 'commit' for iot-hub-device-update here
 adu_git_branch=develop
-adu_git_commit=e099aaca0ad2b8849de24a13b1aa8e3ddf009251
+adu_git_commit=4794e82716f0a3dd0d2ff441a114a4788bd09d01
```

Comment:
yes, it would eventually. I'm thinking after a release of agent to main, we would change this here to a commit on main and then this kirkstone branch into main.